apps/editor.planx.uk/src/@planx/components/Checklist/model.ts

```diff
@@ -225,7 +113,12 @@ export const validationSchema = baseNodeDataValidationSchema.concat(
     })
     .test({
       name: "atLeastOneDataField",
-      test: function ({ fn, options = [], groupedOptions = [] }) {
+      test: function (value) {
+        const {
+          fn,
+          options = [],
+          groupedOptions = [],
+        } = value as ChecklistWithOptions;
```

Comment:
Casting is required here - yup isn't correctly inferring the full type based on the `.concat()` operation.

It's likely that many of these tests will need to be shared once `ResponsiveChecklist` is implemented.